packages/explorer/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@latticexyz/explorer",
+  "version": "2.1.0",
+  "private": true,
+  "description": "Explorer is a tool for visually exploring and manipulating the state of worlds",
+  "bin": {
+    "explorer": ".next/standalone/packages/explorer/server.js"
+  },
+  "scripts": {
+    "build": "next build && cp -r .next/static .next/standalone/packages/explorer/.next && pnpm link .",
+    "dev": "next dev",
+    "lint": "next lint",
+    "start": "node .next/standalone/packages/explorer/server.js"
+  },
+  "dependencies": {
+    "@hookform/resolvers": "^3.9.0",
+    "@latticexyz/common": "workspace:*",
+    "@latticexyz/protocol-parser": "workspace:*",
+    "@latticexyz/schema-type": "workspace:*",
+    "@latticexyz/store": "workspace:*",
+    "@latticexyz/world": "workspace:*",
+    "@radix-ui/react-checkbox": "^1.1.1",
+    "@radix-ui/react-dropdown-menu": "^2.1.1",
+    "@radix-ui/react-label": "^2.1.0",
+    "@radix-ui/react-select": "^2.1.1",
+    "@radix-ui/react-separator": "^1.1.0",
+    "@radix-ui/react-slot": "^1.1.0",
+    "@radix-ui/react-toast": "^1.2.1",
+    "@radix-ui/themes": "^3.0.5",
+    "@tanstack/react-query": "^5.51.3",
+    "@tanstack/react-table": "^8.19.3",
+    "@uidotdev/usehooks": "^2.4.1",
+    "@wagmi/core": "^2.12.1",
+    "better-sqlite3": "^8.6.0",
```

Comment:
haven't checked but we should make sure these common dep versions are aligned with the rest of our packages (wagmi/viem used in templates, sqlite in indexer, react in templates and react package, etc.)

Reply:
Found and updated these disrepancies: `@types/node`, `@types/react`, `@types/react-dom`. Also removed `prettier` as it was in root `package.json` already.